hooks/useVotesForAddress.ts

```diff
@@ -0,0 +1,11 @@
+import useSWR from 'swr';
+import { useVoteDirections } from './useVoteDirections';
+import { Vote } from '../types/Vote';
+
+export function useVotesForAddress(address: string) {
```

Comment:
does this get used?

Reply:
good catch was replaced by the paginated voter lists, can remove. Might be useful for tooltips / summary views but this is also a very easy add back when needed

Reply:
noting for follow up